### PR TITLE
aurora 0.20.0

### PR DESCRIPTION
**Pull request by thinker0**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by thinker0** [comment]
@mikemcquaid

File: Formula/aurora-cli.rb [hunk 1]
```diff
@@ -11,17 +11,16 @@ class AuroraCli < Formula
...
   end
 
-  if MacOS.version == :high_sierra
+  depends_on "python@2" if MacOS.version <= :snow_leopard
+
+  if MacOS.version == :sierra || MacOS.version == :el_capitan
     patch do
-      url "https://raw.githubusercontent.com/Homebrew/formula-patches/21a7e7d/aurora-cli/pants_version_1.4.0_dev20.diff"
-      sha256 "0f4e3dcab78974d43ff5225df68969609587656313f8e495908523f89f6cb0a7"
+      url "https://raw.githubusercontent.com/thinker0/aurora/virtualenv-update/pants.diff"
```
Comment (fxcoudert) [inline Formula/aurora-cli.rb]: Why is the patch needed? Has the bug been reported to upstream?

Comment (thinker0) [reply Formula/aurora-cli.rb]: https://issues.apache.org/jira/browse/AURORA-1984

Report Upstream

**Comment by thinker0** [comment]
@fxcoudert
Hele me this file ?? T.T

**Pull request closed without merging**